### PR TITLE
Enable opensearch

### PR DESCRIPTION
**Pull request by vetharupini**
Enable opensearch indexer 
Added test cases for Opensearch and elasticsearch indexer

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Default
 FILE_DOWNLOAD_DIR_PATH=
-EXTRACTED_DIR_PATH=
+EXTRACTED_DIR_PATH="/home/vetharupini/Downloads/1q23_earnings_press_release_with_tables.pdf.json"
```
Comment (SyedHabibM) [inline .env:3]: Remove this

File: docs/source/vector_database/opensearch.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
+.. _opensearch:
+
+OpenSearch
+===========
+
+.. image:: https://github.com/decisionfacts/semantic-ai/blob/master/docs/source/_static/images/logo/opensearch_logo_img.png?raw=true
+    :alt: Logo
```
Comment (SyedHabibM) [inline docs/source/vector_database/opensearch.rst:7]: Align the docs

**Review by SyedHabibM — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (vetharupini) [reply .env]: solved.

File: docs/source/vector_database/opensearch.rst (hunk 2 above)
Comment (vetharupini) [reply docs/source/vector_database/opensearch.rst:7]: solved

**Review by SyedHabibM — APPROVED** [review]

**Pull request merged**